routes/routes.go

```diff
@@ -0,0 +1,55 @@
+package routes
+
+import (
+	"github.com/gofiber/fiber"
+	"learning_go/book"
+	"learning_go/database"
+)
+
+func GetBooks(c *fiber.Ctx) {
+	db := database.DB.Db
+	var books []book.Book
+	db.Find(&books)
+	c.JSON(books)
+}
+
+func GetBook(c *fiber.Ctx) {
+	id := c.Params("id")
+	db := database.DB.Db
+	var book book.Book
+	db.Find(&book, id)
+
+	if book.Name == "" {
+		c.Status(404).Send("No book found with the ID" + id)
+		return
+	}
+
+	c.JSON(book)
+}
+
+func AddBook(c *fiber.Ctx) {
+	db := database.DB.Db
+	book := new(book.Book)
+
+	if err := c.BodyParser(book); err != nil {
+		c.Status(500).Send("Unable to parse JSON data. Please review your request body")
```

Comment:
a 500 status code is "our" fault, it means something on our side failed. If it's bad data from them, it should be a 400-level error.